docs/guides/platform/disk-images/managing-the-kernel-on-a-linode/index.md

```diff
@@ -0,0 +1,210 @@
+---
+slug: managing-the-kernel-on-a-linode
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Use the Linode Manager to change which kernel your Linode boots.'
+keywords: ["kernel", "grub"]
+tags: ["linode platform","cloud manager"]
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0/)'
+aliases: ['/tools-reference/custom-kernels-distros/run-a-distribution-supplied-kernel-with-kvm/','/platform/how-to-change-your-linodes-kernel/','/run-a-distribution-supplied-kernel-with-kvm/','/tools-reference/custom-kernels-distros/run-a-distribution-supplied-kernel/','/platform/update-kernel/', '/guides/how-to-change-your-linodes-kernel/']
```

Comment:
I think an alias from the Update Your Linux Kernel guides is missing here: `/guides/update-kernel/`